python/astro_metadata_translator/translators/decam.py

```diff
@@ -393,6 +393,35 @@ def fix_header(
             modified = True
             log.debug("%s: Set FILTER to %s because OBSTYPE is %s", log_label, header["FILTER"], obstype)
 
+        # The following provides corrections to the boresight header info for
+        # the raw data files obtained from http://astroarchive.noirlab.edu/,
+        # program number 2013A-0351, of the Trifid and Lagoon Nebulae region.
+        if "Trifid" in header.get("OBJECT", "unknown") and obstype == "object":
+            date_obs = astropy.time.Time(header["DATE-OBS"], format="isot", scale="utc")
+            date_min = astropy.time.Time("2013-02-10", format="isot", scale="utc")
+            date_max = astropy.time.Time("2013-02-14", format="isot", scale="utc")
+            if date_obs > date_min and date_obs < date_max:
+                delta_ra = -0.027417  # median diff between header RA & https://astroarchive.noirlab.edu RA
+                delta_dec = -0.002833  # median diff between header Dec & https://astroarchive.noirlab.edu DEC
+                log.debug("Found Trifid in OBJECT header.  Shifting TELRA and TELDEC...")
+                radec_target = SkyCoord(
+                    header["RA"],
+                    header["DEC"],
+                    frame=header["RADESYS"].strip().lower(),
+                    unit=(u.hourangle, u.deg),
+                )
+                radec_fixed = SkyCoord(
+                    radec_target.ra.deg + delta_ra,
+                    radec_target.dec.deg + delta_dec,
+                    frame=header["RADESYS"].strip().lower(),
+                    unit=(u.deg, u.deg),
+                )
+                log.debug("Orig TELRA = {} TELDEC = {}".format(header["TELRA"], header["TELDEC"]))
+                header["TELRA"] = radec_fixed.ra.hourangle
+                header["TELDEC"] = radec_fixed.dec.deg
+                modified = True
+                log.debug("New TELRA = {:.5f} TELDEC = {:.5f}".format(header["TELRA"], header["TELDEC"]))
```

Comment:
(1) For debugs please don't pre-format the log message and instead use `log.debug("something %f", telra)` style.
(2) Please combine into a single log message rather than 3.